packages/twenty-front/src/modules/object-record/record-show/record-detail-section/components/RecordDetailRelationSection.tsx

```diff
@@ -142,7 +122,11 @@ export const RecordDetailRelationSection = ({
 
   const showContent = () => {
     if (loading) {
-      return <StyledRecordDetailRelationSectionSkeletonLoader />;
+      return (
+        <RecordDetailRelationSectionSkeletonLoader
+          isPeopleField={fieldName === 'people'}
```

Comment:
why do we need to know if People? I feel it's not the right rule, what's the intent?